src/planning/plan/tests/plan-consumer-test.ts

```diff
@@ -72,7 +107,7 @@ recipe
       assert.isEmpty(consumer.getCurrentSuggestions());
 
       // Updates suggestions.
-      await storeResults(consumer, helper.findSuggestionByParticleNames(['ItemMultiplexer', 'List']));
+      await storeResults(consumer, findSuggestionByParticleNames(suggestions, ['ItemMultiplexer', 'List']));
```

Comment:
in this particular case, there is only one suggestions, so it's easier to just verify the suggestions.length and the suggestions[0]'s particle names, no need for the helper method.